lib/beaker-pe/pe-client-tools/install_helper.rb

```diff
@@ -0,0 +1,74 @@
+module Beaker
+  module DSL
+    module InstallUtils
+      module PEClientTools
+
+        def install_pe_client_tools_on(hosts, opts = {})
+          # FIXME: accomodate production released location(s)
+          #{{{
+          product = 'pe-client-tools'
+          required_keys = [:puppet_collection, :pe_client_tools_sha, :pe_client_tools_version]
+
+          unless required_keys.all? { |opt| opts.keys.include?(opt) }
+            raise ArgumentError, "The keys #{required_keys.to_s} are required in the opts hash"
+          end
+          urls = { :dev_builds_url   => "http://builds.delivery.puppetlabs.net",
+          }
+
+          opts = urls.merge(opts)
+
+          block_on hosts do |host|
+            package_name = nil
+            variant, version, arch, codename = host['platform'].to_array
+            case host['platform']
+            when /el-|fedora|sles|centos|cisco_/
+              release_path = "#{opts[:dev_builds_url]}/#{product}/#{ opts[:pe_client_tools_sha] }/artifacts/#{variant}/#{version}/#{opts[:puppet_collection]}/#{arch}"
```

Comment:
looks like
```ruby
"#{opts[:dev_builds_url]}/#{product}/#{ opts[:pe_client_tools_sha] }/artifacts/
```
is duplicated across all of the `release_path` lines for each of these platforms. It seems like we should be able to pull that out to make these bodies easier to read, since they'd have less duplication.